pkg/apis/core/validation/validation.go

```diff
@@ -2896,6 +2896,14 @@ func ValidatePodSpec(spec *core.PodSpec, fldPath *field.Path) field.ErrorList {
 		allErrs = append(allErrs, field.Forbidden(fldPath.Child("priority"), "Pod priority is disabled by feature-gate"))
 	}
 
+	if spec.ShareProcessNamespace != nil {
```

Comment:
set to nil in `pkg/api/pod#DropDisabledAlphaFields`

Reply:
Done